project/plugins.sbt

```diff
@@ -4,7 +4,7 @@ resolvers += Resolver.typesafeRepo("releases")
 
 addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "3.24.0")
 addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "2.6.0")
-addSbtPlugin("org.playframework" % "sbt-plugin"         % "3.0.8")
+addSbtPlugin("org.playframework" % "sbt-plugin"         % "3.0.7")
 addSbtPlugin("org.scoverage"     % "sbt-scoverage"      % "2.3.1")
 addSbtPlugin("org.scalameta"     % "sbt-scalafmt"       % "2.4.6")
 addSbtPlugin("org.typelevel"     % "sbt-tpolecat"       % "0.5.2")
```

Comment:
Can you also try to update to latest versions? The Scala 3 migration is a good opportunity to use latest versions of plugins and dependencies.